day1.md

```diff
@@ -0,0 +1,119 @@
+
+We were pretty excited when we finally arrived and gonna spend the rest of the day in door!
+<br/><img src="day1/14434912_1115747191808307_2336258069687317341_o.jpg" width="500"><br/>
+
+The first thing we needed to do, after checking-in, was to figure out our sizes and styles for Yukatas. We were gonna wear them the whole time hanging in Odaiba Onsen.
+
+Odaiba Onsen was huge inside! Here's the [floor map](https://jonellepatrick.com/oedo-onsen-2/).
+<br/><img src="day1/oedoonsenmap.jpg" width="500"><br/>
+
+We started by hanging out in "Town Square":
+>[which has been built to resemble an Edo-era town. Lining the “street” are many shops, food stands and restaurants – udon and soba noodles, sushi, unagi (eel), oden, beer, sake, etc.](https://jonellepatrick.com/oedo-onsen-2/)
+
+<br/><img src="day1/IMG_20160905_161521.jpg" width="500">
+<br/><img src="day1/IMG_20160905_200120.jpg" width="500"><br/>
+
+Matt and Katherine got crepes and I got a huge takoyaki:
+<br/><img src="day1/14444593_1115749701808056_2846704788163472424_o.jpg" width="500">
+<br/><img src="day1/IMG_20160905_161923.jpg" width="500">
+<br/><img src="day1/IMG_20160905_161930.jpg" width="500">
+
+There were plenty of dinning tables:
+<br/><img src="day1/IMG_20160905_162618.jpg" width="500">
+
+and small shops/game booths:
+<br/><img src="day1/IMG_20160905_162317.jpg" width="500">
+
+I tried out this traditional game I saw a lot on animations/TVs!
+<br/><img src="day1/IMG_20160905_163059.jpg" width="500">
+
+Then we headed to the outdoor foot bath area:
+<br/><img src="day1/14379918_1115751508474542_3542901742528241190_o.jpg" width="500"><br/>
+Matt & Kath!
+<br/><img src="day1/IMG_20160905_170119.jpg" width="500">
+
+[Fish Therapy](https://jonellepatrick.me/2013/04/08/weirdest-beauty-treatment-ever/)<sup>[5](#fish_therapy)</sup>, where you let these small fishes to clean your feet by nibbling!
+<br/><img src="day1/14424770_1115750975141262_9061303587866261300_o.jpg" width="500">
+<br/><img src="day1/14409904_1115750858474607_1375099417498840859_o.jpg" width="500">
+
+We also made our first friend in Japan, Hoang, who was next to me while my feet getting nibbled and was also from the Bay Area!
+
+Having enough fun outside, it was onsen<sup>[6](#onsen)</sup> time!
+
+It was my first time trying out onsen and it was a lot of fun! It's pretty relaxing and quite liberating 😂
+Kath and I tried out all different bathes. I always saw people scrubbing each others' backs during onsens in Japanese animations/TVs, so for the sake of getting a full authentic onsen experience, I decided to give it a try. And man, that was quite an experience. It's like getting a body massage while seeing your dead skin coming off. It's quite refreshing as well, because I basically just laid there for 45 mins.
```

Comment:
"baths" instead of "bathes"
"each other**'s**" instead of "each others'"
more of nitpick: "TV shows" instead of "TVs"